Modules/DataModel/Sources/PocketCastsDataModel/Private/Managers/Util/DatabaseHelper.swift

```diff
@@ -656,6 +656,16 @@ class DatabaseHelper {
             }
         }
 
+        if schemaVersion < 43 {
```

Comment:
Ideally, we would put this behind a feature flag. However, for deselecting chapters, we have a remote feature flag, so this is not possible. We could, for example, run into the flag being enabled after migration has happened and the feature would be enabled without its corresponding database change.

Since this is a simple column addition, I think it's safe to have it.